handler/models.go

```diff
@@ -0,0 +1,39 @@
+package handler
+
+import "time"
+
+//struct for eventhandler which handles the instance and the start of the api
```

Comment:
Comments should start with the name of the exported element : `//EventReport handles the....`